README.md

```diff
@@ -129,6 +129,18 @@ gauge.get({ room: 'kitchen' })
 # => 21.534
 ```
 
+Also you can use gauge as the bi-directional counter:
+
+```ruby
+gauge = Prometheus::Client::Gauge.new(:concurrent_requests_total, '...')
+
+gauge.increment({ service: 'foo' })
```

Comment:
Hmm, I give it the same names as in `Counter` (it have method called `increment` too, which apparently doesn't conform to spec, but it's written 4 years ago, much before the spec, I think)